views/home.handlebars

```diff
@@ -1,54 +1,11 @@
-<h1 class="text-break text-center">MySpot - Homepage</h1>
-<h1><div class="container-sm">
-     <div class="text-center">
-        {{!-- {{#if matchesSessionUser}}
-            
-        {{/if}} --}}
-            <button type="button" class="btn mx-auto btn-outline-danger logout-button" href='/home'>Home</button>
-            {{#if isLoggedIn}}
-            <button type="button" class="btn mx-auto btn-outline-danger logout-button" href='/newspot'>+ New Spot</button>
-            <button type="button" class="btn mx-auto btn-outline-danger logout-button" href='/profile'>My Spots</button>
-            <button type="button" class="btn mx-auto btn-outline-danger logout-button" href='/logout'>Logout</button>
-            {{/if}}
-     </div>
-     
-      
-    <div class="row justify-content-center">
-        <div class="col-sm-12 col-lg-6 my-2">
-            <div class="card">
-                <div class="card-header">
-                  {{!random Spot}}
-                </div>
-                <div class="card-body">
-                    <h5 class="card-title">{{spot.title}}</h5>
-                    <p class="card-text">{{spot.description}}</p>
-                    <a href="/spot/{{spot.id}}" class="btn btn-primary">About this Spot:</a>
-                   
-                    
-                </div>
-            </div>
-          <h5 class= "text-center"><br>Spot pinned by: {{spot.User.username}}</h5>
-        </div>  
-    </div></h1>
-
-    <h4 class="text-center">X Coordinates: {{spot.longitude}}&nbsp;&nbsp;&nbsp;||&nbsp;&nbsp;&nbsp; Y Coordinates: {{spot.latitude}}<br></h4>
-
-      <div class="card mx-auto" style="width: 18rem;"> 
-        <div class="card-body">
-          <h5 class="card-title">Description: </h5>
-          <p class="card-text">{{spot.description}}</p>
-        </div>   
-      </div>
-
-      <div class="card mx-auto" style="width: 18rem;">
-        <div class="card-body">
-          <h5 class="card-title">Tags: </h5>
-          <p class="card-text">{{spot.tags}}</p>
-        </div>    
-      </div>   
-   </div> 
-
-
+<h1>Home</h1>
```

Comment:
Your code is not up to date with the main branch. Please make sure you do a git pull on main and merge main with your branch.